### PR TITLE
Functioning Plus/Cross tab buttons 

### PR DESCRIPTION
**Pull request by Karyum**
related to #75

### CODE REVIEW

File: app/public/index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h3 class="nugget-title">
     <!-- ADD-FORM -->
 
     <!-- LOCATION-SELECT -->
-    <div class="location-select-display">
+    <div class="location-select-display not-visible">
```
Comment (mattlub) [inline app/public/index.html]: you don't need a not-visible class (you don't use it) here and line 27

File: app/public/map.js [hunk 2]
```diff
@@ -97,4 +97,18 @@ var nazarethNuggets = (function () { // eslint-disable-line
   centerButton.addEventListener('click', function (e) {
     mymap.locate({setView: true})
   })
+
+  var locationSelector = document.querySelector('.location-select-display')
```
Comment (mattlub) [inline app/public/map.js]: why not call this locationSelectDisplay?

**Review by Suuuuha — APPROVED** [review]

File: app/public/css/main.css [hunk 3]
```diff
@@ -92,10 +92,10 @@ html, body {
 /*
 4) NUGGET-INFO
 */
-.nugget-info-tab {
-  display: none;
+.nugget-info-tab.visible {
```
Comment (mattlub) [inline app/public/css/main.css:95]: aren't we having the animation where the tab scrolls up when it becomes visible?

File: app/public/map.js [hunk 4]
```diff
@@ -97,4 +97,18 @@ var nazarethNuggets = (function () { // eslint-disable-line
...
+  var addNuggetButton = document.querySelector('.add-nugget-button')
+  addNuggetButton.addEventListener('click', function (e) {
+    locationSelector.classList.toggle('visible')
+  })
+
+  // tic and cross goes here mario
```
Comment (mattlub) [inline app/public/map.js]: better comment?

**Review by mattlub — CHANGES_REQUESTED** [review]

File: app/public/map.js (hunk 2 above)
Comment (Karyum) [reply app/public/map.js]: because thats how it is called on suha's pull reuqest

File: app/public/map.js (hunk 4 above)
Comment (Karyum) [reply app/public/map.js]: ohh didn't see that thanks :wink:

File: app/public/css/main.css (hunk 3 above)
Comment (Karyum) [reply app/public/css/main.css:95]: i don't understand whats the question here

File: app/public/index.html (hunk 1 above)
Comment (Karyum) [reply app/public/index.html]: cheers!

File: app/public/index.html [hunk 5]
```diff
@@ -24,7 +24,7 @@ <h1 class="app-title">City Nuggets</h1>
     <img class="center-button" src="./assets/re-center.png">
 
     <!-- NUGGET-INFO -->
-    <div class="slide-up-tab nugget-info-tab">
+    <div class="slide-up-tab nugget-info-tab not-visible">
```
Comment (mattlub) [inline app/public/index.html]: not-visible is not used

**Review by mattlub — CHANGES_REQUESTED** [review]

**Review by mattlub — APPROVED** [review]

**Pull request merged**